bin/check-fail.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -e
+
+if [ "$TRAVIS_PULL_REQUEST" = false ]
+then
+  echo "Skipping $0 for non-PR build"
+  exit 0
+fi
+
+TARGET_BRANCH="$TRAVIS_BRANCH"
+if echo "$TRAVIS_PULL_REQUEST_BRANCH" | grep -qi refactor
+then
+    echo "The title of the PR indicates this is a refactoring; skipping this check"
+    exit 0
+fi
+
+if [ x"$TARGET_BRANCH" = x ]
+then
+    echo "No target branch found"
+    exit 1
+fi
+
+git checkout "$TARGET_BRANCH"
+
+for d in t tests test spec
+do
+    git checkout HEAD@{1} -- $d 2> /dev/null || true
+done
+
+if bundle exec rake test
+then
+    echo "Your newly introduced tests should have failed on $TARGET_BRANCH"
```

Comment:
Ooh, yep, good idea! Have added some extra info in https://github.com/everypolitician/scraped/pull/67/commits/65166c49a993da08eb144ea963869429ae6e54ba.